docs/content/foundations/layout-positioning/layout-grid.mdx

```diff
@@ -0,0 +1,356 @@
+
+<GridColExample />
+
+```html
+<div className="grid-row">
+	<div className="grid-col-8 grid-offset-4">.grid-col-8.grid-offset-4</div>
+</div>
+```
+
+## Column Wrapping
+
+Items wrap when the sum of the grid columns is more than 12 as shown in the following example.
+
+```html
+<div className="grid-row">
+	<div className="grid-col-8">.grid-col-8</div>
+	<div className="grid-col-3">.grid-col-3</div>
+	<div className="grid-col-5">.grid-col-5</div>
+</div>
+```
+
+<h2 id="gutters">Gutters</h2>
+
+### Default Gutter
+
+Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the row. The default gap width is 2 units and 4 units at `desktop` and higher. Customize the width of the gap by adjusting the value of `$theme-column-gap` in project settings as shown in the following example.
+
+<GridColExample />
+
+```html
+<div className="grid-row grid-gap">
+	<div className="grid-col-4"><div>.grid-col-4</div></div>
+	<div className="grid-col-4"><div>.grid-col-4</div></div>
+	<div className="grid-col-4"><div>.grid-col-4</div></div>
+</div>
+```
+
+### Large Gutter
+
+`.grid-gap-lg` adds a larger gap (or gutter) between each column in a row than `grid-gap`. The default large-gap width is 32px (4 spacing units). Customize the width of the large gap by adjusting the value of `$theme-column-gap-lg` in project settings. There is also a `.grid-gap-sm` (2px) set with `$theme-column-gap-sm`. Also, you can add the following system values with `grid-gap`:
+
+- `.grid-gap-2px`
+- `.grid-gap-05`
+- `.grid-gap-1`
+- `.grid-gap-2`
+- `.grid-gap-3`
+- `.grid-gap-4`
+- `.grid-gap-5`
+- `.grid-gap-6`
+
+<GridColExample />
+
+```html
+<div className="grid-row grid-gap-lg">
+	<div className="grid-col-4"><div>.grid-col-4</div></div>
+	<div className="grid-col-4"><div>.grid-col-4</div></div>
+	<div className="grid-col-4"><div>.grid-col-4</div></div>
+</div>
+```
+
+## Sass Mixins
+
+When generating your CSS from USWDS source files, you have the option of customizing many system defaults by modifying project theme variables. USWDS also provides grid mixins for adding grid functionality to custom semantic component CSS.
+
+### Variables
+
+Variables and maps determine the number of columns, gutter width, and media-query point at which to begin floating columns. We use variables to generate the predefined grid classes documented on this page, as well as for the custom mixins noted under Spacing settings and Utilities settings.
+
+#### Spacing Settings
+
+```
+// Values are set as units tokens.
+$theme-column-gap-sm:               2px;
+$theme-column-gap-md:               2;
+$theme-column-gap-lg:               3;
+$theme-column-gap-mobile:           2;
+$theme-column-gap-desktop:          4;
+$theme-grid-container-max-width:    'widescreen';
+$theme-site-margins-breakpoint:     'desktop';
+$theme-site-margins-width:          4;
+$theme-site-margins-mobile-width:   2;
+```
```

Comment:
Let's remove this section because we do not support changing the default values.